presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Send proper error response to client for certain types of errors during query
```

Comment:
Hm. I hope we now send proper error response for _all_ types of errors, not just _certain types_.
-- Since this sentence does not contain "Fix", it can be misunderstood. I think i understand this only because I know the related PR.

Reply:
Agreed, this needs further re-wording.